blockdevice/blockdevice.go

```diff
@@ -36,6 +36,22 @@ type BlockDevice struct {
 	// BlockDevice if it exists
 	FSInfo FileSystemInformation
 
+	// Parent is the parent device of this blockdevice, if it exists.
```

Comment:
Will it be possible to put some more clarity. Perhaps a yaml corresponding to this struct that maps to the BlockDevice?
I understand this is not a yaml/json. However, I felt it will be easier to understand.

Reply:
Sample blockdevice structs has been added to the comment section to make it easier to understand how the various fields are used.